### DISCUSSION TITLE
How to get all markets of event?

### DISCUSSION THREAD

**Discussion by San10k** [cloudbet/docs#27] [Q&A] [answered]
When I was make requests by Competition or Eventid, in Dota2 events, in responce get only with 4 markets: dota_2.correct_score_in_maps, dota_2.map_handicap, dota_2.total_maps and dota_2.winner.
Any ways to get dota_2.map_winner and other markets?

**Comment by kgravenreuth** [comment]
@San10k this looks like a bug to me as the dota2 map winner should be there as per our [API markets list](https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json) . We'll investigate and update here afterwards.

**Comment by kgravenreuth** [comment, marked as answer]
@San10k we've added some missing markets to the list now, including `dota_2.map_winner_v2` . These are now available in the Cloudbet API. 

Latest Market list here as always: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json

Please let me know if you still have any questions.
